deker_server_adapters/collection_adapter.py

```diff
@@ -57,12 +58,13 @@ def read(self, name: str) -> dict:
 
         :param name: Name of collection
         """
-        response = self.client.get(f"/{self.COLLECTION_URL_PREFIX}/{name}")
+        url = f"/{self.COLLECTION_URL_PREFIX}/{name}"
+        response = make_request(url=url, nodes=self.hash_ring.nodes, client=self.client)
 
-        if response.status_code == STATUS_OK:
+        if response and response.status_code == STATUS_OK:
```

Comment:
Is there a possibility of getting no response at all?

Reply:
Yes, there could be an exception. I'll add handler